.buildkite/hooks/pre-command

```diff
@@ -55,6 +55,15 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "fleet-server" && "$BUILDKITE_STEP_KEY" == "
   export EC_API_KEY=${EC_API_KEY_SECRET}
 fi
 
+# BK analytics
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "fleet-server" ]]; then
+  if [[ "$BUILDKITE_STEP_KEY" == "int-test" || "$BUILDKITE_STEP_KEY" == "e2e-test" ]]; then
+    echo "--- Prepare BK test analytics token :vault:"
+    BUILDKITE_ANALYTICS_TOKEN=$(vault kv get -field token kv/ci-shared/platform-ingest/buildkite_fleet_server_analytics_token)
```

Comment:
I have no comment; we did in other repositories, but IMHO, it's not ideal either—except Beats, due to the nature of the pipelines.

The secret should be in a repository-scoped vault path rather than a shared vault path. This is a good idea because the secret makes sense only for this repository.

Reply:
Do we have repo-scoped vault paths that we use for CI?

Reply:
sure we do https://docs.elastic.dev/ci/using-secrets#using-repository-scoped-secrets

But we can move them afterwards.